app/screens/channel_info/channel_info.js

```diff
@@ -234,58 +228,92 @@ export default class ChannelInfo extends PureComponent {
         );
     });
 
-    handleDeleteOrLeave = preventDoubleTap((eventType) => {
+    handleLeave = preventDoubleTap(() => {
+        const title = {id: t('mobile.channel_info.alertTitleLeaveChannel'), defaultMessage: 'Leave {term}'};
+        const message = {
+            id: t('mobile.channel_info.alertMessageLeaveChannel'),
+            defaultMessage: 'Are you sure you want to leave the {term} {name}?',
```

Comment:
I can see how `term` is being set in `alertAndHandleYesAction` but the same is not true for `name`

Reply:
i believe its set in line 324